launcher/ui/pages/global/APIPage.cpp

```diff
@@ -79,6 +82,20 @@ void APIPage::applySettings()
     s->set("PastebinURL", pastebinURL);
     QString msaClientID = ui->msaClientID->text();
     s->set("MSAClientIDOverride", msaClientID);
+    QUrl metaURL = ui->metaURL->text();
+    if(metaURL.isEmpty())
+    {
+        s->set("MetaURL", BuildConfig.META_URL);
+        return;
+    }
```

Comment:
Don't like this. Maybe we want to do this like with MSA client ID instead? We register a setting called `MetaURLOverride` and only use it if it is present. If it is empty, we just use the value from BuildConfig.

Reply:
Ok